storage/migrations/5_windowpost.go

```diff
@@ -0,0 +1,24 @@
+package migrations
+
+import (
+	"github.com/go-pg/migrations/v8"
+)
+
+// Schema version 5 provides additional tracking of the miner->deadline->partition->sector relation
+
+func init() {
+
+	up := batch(`
+	CREATE TABLE IF NOT EXISTS "miner_sector_posts" (
+		"miner_id" text not null,
+		"sector_id" int not null,
+		"epoch" int not null,
```

Comment:
`bigint`

Reply:
I suggested he correct this to `int`, you're right though.

Reply:
I suggested he correct this to `int`, you're right though.